### ISSUE TITLE
Oceananigans.jl ocean modeling example

### ISSUE THREAD

**Issue by hdrake** [ClimateMARGO/interactive#7]
@navidcy and @glwagner, I'd like to include an Oceananigans.jl example of an eddying double gyre in our curriculum of reactive Pluto.jl notebooks (see [our current example](https://margo.plutojl.org/introduction.html)). In the Ferrari group meeting today, we discussed making an outreach demo that would allow students to release little plastic particles in a double gyre and watch them accumulate in the convergence zones (garbage patches).

Would it make sense to use output from @navidcy's double-gyre example (https://github.com/CliMA/Oceananigans.jl/pull/1085)? An alternative would be to just use a much cheaper barotropic model (i.e. nz=1) and somehow parameterize the effect that Ekman convergence has on the plastics.

What do you think?

**Comment by navidcy** [comment]
sounds great!
the example in https://github.com/CliMA/Oceananigans.jl/pull/1085 is a bit expensive to run... so I'm not sure you can run it interactively on a Pluto notebook... Perhaps if we run it to equilibration then you can checkpoint it from an equilibrated state and run in the notebook for 1-2 years (with particles)? We can try!

**Comment by navidcy** [comment]
These interactive notebooks look great btw!

**Comment by hdrake** [comment]
> Perhaps if we run it to equilibration then you can checkpoint it from an equilibrated state and run in the notebook for 1-2 years (with particles)?

Yeah, that could work! Alternatively, we could just load in some pre-computed outputs (e.g. 100 snapshots from the statistical steady state) and linearly interpolate (and loop) the velocities in time for the Lagrangian integrations, but maybe that is less fun.

**Comment by glwagner** [comment]
I think we should try integration offline first since I think that's most likely to be reponsive and snappy in a Pluto notebook --- what do you think @navidcy?

We can advect particles with the surface field only (whether we have `Nz=1` or more levels)?
